src/ts/components/subtitlesettings/fontfamillyselectbox.ts

```diff
@@ -0,0 +1,81 @@
+import {SelectBox} from '../selectbox';
+import {ListSelectorConfig} from '../listselector';
+import {UIInstanceManager} from '../../uimanager';
+import {SubtitleOverlay} from '../subtitleoverlay';
+import {StorageUtils} from '../../utils';
+
+/**
+ * A select box providing a selection of different font familly.
+ */
+export class FontFamillySelectBox extends SelectBox {
+
+  private overlay: SubtitleOverlay;
+
+  private fontFamilly: { [key: string]: string} = {
```

Comment:
Consolidate `fontFamily`, `fontVariant` and `fontStyle` into a single object. 3 separate objects with indirectly associated lists make maintenance unnecessarily complicated.

```ts
interface FontFamily {
  name: string;
  family: string;
  variant: string;
  style: string;
}

private fontFamilies: FontFamily[] = [{
  name: 'default',
  family: 'default',
  variant: 'unset',
  style: 'unset',
}, {
  name: 'monospaced serif',
  family: '"Courier New",Courier,"Nimbus Mono L","Cutive Mono",monospace',
  variant: 'unset',
  style: 'unset',
}, ... ];
```